dotnet/src/SemanticKernel.Abstractions/Kernel.cs

```diff
@@ -114,6 +114,9 @@ public sealed class Kernel
             FunctionInvoked = this.FunctionInvoked,
             PromptRendering = this.PromptRendering,
             PromptRendered = this.PromptRendered,
```

Comment:
Unrelated to this PR, but can we get rid of the events now?

Reply:
Good question :) I actually did it during filters v2 implementation, but then we decided to temporarily revert them back. I think the decision was to remove them when we graduate filters to be non-experimental.